ncloud/resource_ncloud_public_ip.go

```diff
@@ -167,9 +170,20 @@ func resourceNcloudPublicIpUpdate(d *schema.ResourceData, meta interface{}) erro
 		}
 
 		if len(n.(string)) > 0 {
-			if err := associatedPublicIp(d, config); err != nil {
+			if err := resource.Retry(d.Timeout(schema.TimeoutDelete)-time.Minute*time.Duration(19), func() *resource.RetryError {
+				if err := associatedPublicIp(d, config); err != nil {
+					errBody, _ := GetCommonErrorBody(err)
+					if errBody.ReturnCode == "1003016" {
+						time.Sleep(1)
```

Comment:
```suggestion
						time.Sleep(time.Second * 1)
```
`time.Sleep()`의 인자인 `Duration`의 default 단위가 nanoseconds라서 위 와 같이 1초의 interval을 주고 싶으시면 위와 같이 수정 부탁드립니다~ `(time.Second * second)`

- go time package
  - https://github.com/golang/go/blob/ba1ef54c1ee0e5ed8fd572dbb4ec67548cd5a55e/src/time/time.go#L610-L617